### ISSUE TITLE
Execute Cura 12.09 under Osx 10.7

### ISSUE THREAD

**Issue by darkomen** [daid/LegacyCura#217]
Hi i follow the steps in your tutorial (http://daid.github.com/Cura/?page=MacOS ) I install wxtools, PyOpenGL, and all tools you mentioned in tutorial.
I downloaded all repositorio and execute the package.sh file, when its done, i go to folder osx64-Cura-12.09 and execute cura.command and i get the follow error:

raceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/wxPython-unicode-2.8.12.1/lib/python2.7/site-packages/wx-2.8-mac-unicode/wx/**init**.py", line 45, in <module>
    from wx._core import *
  File "/usr/local/lib/wxPython-unicode-2.8.12.1/lib/python2.7/site-packages/wx-2.8-mac-unicode/wx/_core.py", line 4, in <module>
    import _core_
ImportError: dlopen(/usr/local/lib/wxPython-unicode-2.8.12.1/lib/python2.7/site-packages/wx-2.8-mac-unicode/wx/_core_.so, 2): no suitable image found.  Did find:
    /usr/local/lib/wxPython-unicode-2.8.12.1/lib/python2.7/site-packages/wx-2.8-mac-unicode/wx/_core_.so: no matching architecture in universal wrapper
Requires wx. Download and install (the Cocoa/64-bit variant) from:
 http://www.wxpython.org/download.php
logout

But, i already install de wxpython... 
some ideas ?
Regards, Santiago

**Comment by mfeherpataky** [comment]
I was there few days ago. 
Do what the error message says: "Requires wx. Download and install (the Cocoa/64-bit variant) from: http://www.wxpython.org/download.php"

Make sure you get the wxPython2.9-osx-cocoa-py2.7 that should get you going.

**Comment by darkomen** [comment]
Thanks!
Now when i execute, i get the follow error:

2012-09-19 10:27:11.561 Python[578:1b03] Persistent UI failed to open file file://localhost/Users/santiagolopezpina/Library/Saved%20Application%20State/org.python.python.savedState/window_1.data: No such file or directory (2)

But the cure.command still working, I was waiting for ten minutes

**Comment by darkomen** [comment]
Before the error of directory, I get this:

```
return _core_.FlexGridSizer_AddGrowableCol(*args, **kwargs)
```

wx._core.PyAssertionError: C++ assertion "!m_cols || idx < (size_t)m_cols" failed at /BUILD/wxPython-src-2.9.4.0/src/common/sizer.cpp(1956) in AddGrowableCol(): invalid column index

i'm thinking this is the problem, I install also the    wxWidgets but don't work :(

**Comment by daid** [comment]
There is no 12.09 release, you downloaded the development branch if you are using 12.09. Which does not work for MacOS right now, which I already knew. Something about some updates I did to the configuration wizard. But the development branch has a few more problems because of changes I did, problems with the printer connection, so I wouldn't recomment using it at all!

The installation instructions tell you to download Cura 12.08 from https://github.com/downloads/daid/Cura/osx64-Cura-12.08.tar.gz
